src/components/sideButtonLink/SideButtonLink.module.scss

```diff
@@ -1,59 +1,59 @@
 .main {
   display: flex;
-  position: fixed;
+  justify-content: center;
+  align-items: center;
   z-index: 3;
-  top: 45%;
-  transform: translateY(-50%);
-  padding: 40px 10px;
+  padding: 10px 10px;
   backdrop-filter: blur(15px);
-  border-radius: 50px;
 
   > span {
     font-size: 14px;
-    margin-top: 5px;
     color: white;
 
     &:last-of-type {
-      margin-top: 4px;
       font-size: 16px;
     }
   }
 
   &:hover {
     transition: all 0.3s;
-    background-color: var(--green-light);
+    background-color: #74ff00;
```

Comment:
why not green light? similar

Reply:
@El-NIVVO asked to change --green-light: #76ff03 to #74ff00;